packages/office-ui-fabric-react/src/components/Button/BaseButton.styles.ts

```diff
@@ -62,16 +62,19 @@ export const getStyles = memoizeFunction((
       }
     ],
 
-    rootDisabled: {
-      backgroundColor: disabledBackground,
-      color: disabledText,
-      cursor: 'default',
-      pointerEvents: 'none',
-      selectors: {
-        ':hover': noOutline,
-        ':focus': noOutline
+    rootDisabled: [
+      getFocusStyle(theme, -1, 'relative', buttonHighContrastFocus),
+      {
+        backgroundColor: disabledBackground,
+        color: disabledText,
+        cursor: 'default',
+        pointerEvents: 'none',
+        selectors: {
+          ':hover': noOutline,
+          ':focus': noOutline
+        }
```

Comment:
Does there need to be any HC styling? (especially for the focused/disabled case in FF)

Reply:
It's not that the styling depends on the disabled flag, it's that HC styling gets implicit disabled styling (especially in the FF case, we may be already be explicitly setting these values when in HC in IE/Edge). Note, we cannot override HC colors in FF so we need to have a story there with representing focus/hover, endabled/disabled, toggled states on the components always so that they will be represented correctly in HC FF (this is the transparent outline approach we've discussed before).

Reply:
Please open a bug for putting in place a solution for HC FF